vignettes/pkgdown.Rmd

```diff
@@ -154,6 +154,8 @@ Once your finalized site is built and published on the web, you should publicize
 
 1. Your repository description on GitHub.
 
+1. The configuration file `_pkgdown.yml`.
```

Comment:
This doesn't feel like the right place to me because it's not about promoting your website, it's about correctly configuring it.

Reply:
moved the reminder